src/state.ml

```diff
@@ -354,6 +354,41 @@ let mtu config session =
       in
       tun_mtu - hdr
 
+let control_mtu config state session =
+  (* we assume to have a tun interface and the server send us a tun-mtu *)
+  let tun_mtu =
+    match Config.find Tun_mtu config with
+    | None -> 1500 (* TODO "client_merge_server_config" should do this! *)
+    | Some x -> x
+  and compress = session.compress in
```

Comment:
Is compression also applied for the control channel?

Reply:
it is not, fixed in 3938ec1f3f4e5c5822bbdca243970f84ab764b75